gwt-localizable-processor/src/test/java/org/gwtproject/i18n/client/ConstantsTest_en.java

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright © 2018 The GWT Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gwtproject.i18n.client;
+
+import static org.junit.Assert.*;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.MissingResourceException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConstantsTest_en {
+
+  @Before
+  public void setUp() throws Exception {
+    System.setProperty("locale", "en");
+  }
+
+  @Test
+  public void testConstantBooleans() {
+    TestConstants types = TestConstants_Factory.create();
+    assertEquals(false, types.booleanFalse());
+    assertEquals(true, types.booleanTrue());
+  }
+
+  @Test
+  public void testConstantDoubles() {
+    TestConstants types = TestConstants_Factory.create();
+    double delta = 0.0000001;
+    assertEquals(3.14159, types.doublePi(), delta);
+    assertEquals(0.0, types.doubleZero(), delta);
+    assertEquals(1.0, types.doubleOne(), delta);
+    assertEquals(-1.0, types.doubleNegOne(), delta);
+    assertEquals(Double.MAX_VALUE, types.doublePosMax(), delta);
+    assertEquals(Double.MIN_VALUE, types.doublePosMin(), delta);
+    assertEquals(-Double.MAX_VALUE, types.doubleNegMax(), delta);
+    assertEquals(-Double.MIN_VALUE, types.doubleNegMin(), delta);
+  }
+
+  @Test
+  public void testConstantFloats() {
+    TestConstants types = TestConstants_Factory.create();
+    double delta = 0.0000001;
+    assertEquals(3.14159f, types.floatPi(), delta);
```

Comment:
this seems riskier to me in jvm vs gwt, since in gwt a float is really a double, but when we get gwt2 and j2cl tests going we'll know more